message/retrylater.go

```diff
@@ -23,18 +24,18 @@ type retryLaterHandler struct {
 
 func (r *retryLaterHandler) Do(ctx context.Context, orig Handler, message *servicebus.Message) Handler {
 	go func() {
-		ctx, span := startSpanFromMessageAndContext(ctx, "go-shuttle.retryLater.Do", message)
+		ctx, span := tracing.StartSpanFromMessageAndContext(ctx, "go-shuttle.retryLater.Do", message)
 		defer span.End()
 
 		select {
-		//TODO this can go past lock duration pretty easily
-		//Ideally we'd also timeout at dequeue time + lockdurtaion - 1 second but don't have access to dequeue time
-		//Maybe we can use context.WithTimeout when we recieve the message?
+		// TODO this can go past lock duration pretty easily
+		// Ideally we'd also timeout at dequeue time + lockdurtaion - 1 second but don't have access to dequeue time
+		// Maybe we can use context.WithTimeout when we recieve the message?
 		case <-ctx.Done():
-			span.AddAttributes(tab.StringAttribute("eventMessage", "Retry context expired"), tab.StringAttribute("eventLevel", "error"))
+			span.Logger().Error(errors.Wrap(ctx.Err(), "Retry context expired"))
```

Comment:
use Logger().Error() which adds the error=true attribute and ensures all errors are following the same attribute list.